packages/twig-new/vite.config.ts

```diff
@@ -0,0 +1,135 @@
+import { defineConfig, BuildOptions, Rollup } from "vite";
+import { viteStaticCopy } from "vite-plugin-static-copy";
+import { OutputOptions, rollup, RollupOptions } from "rollup";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import terser from "@rollup/plugin-terser";
+import swc from "@rollup/plugin-swc";
+import { globSync } from "glob";
+import chalk from "chalk";
+
+import theme from "@ilo-org/themes/tokens/theme/base.json" assert { type: "json" };
+
+const behaviors = globSync("src/components/**/*.behavior.js");
+
+export default defineConfig({
```

Comment:
If we are only using Vite for this plugin, I would rather get rid of it and use only rollup for the build process to simplify things

Reply:
That way you can expose an array of config to rollup, one each for the behavior files and one dummy config that contains the copy plugin.

Reply:
Our build pipeline looks something like this.

![image](https://github.com/user-attachments/assets/15079b03-dc52-424c-9630-0c8bba5ff6ef)


 Since we have the `first phase` that should be triggered before bundling the JS, I think Vite can play a good role there. We can utilize the rich `plugin` system in the future and organize any pipeline operators into plugins. If we ditch `vite` and move to the rollup completely, then we should implement the `glue` step